README.md

```diff
@@ -0,0 +1,335 @@
+# virus-scanner-service
+
+Microservice to perform a virus scan on uploaded files.
+
+This service listens for delta notifications about new files and scans
+those for viruses.
+
+[![Project Status: WIP – Initial development is in progress](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+
+## Getting started
+
+Prerequisites:
+- [file-service](https://github.com/mu-semtech/file-service)
+- [mu-authorization](https://github.com/mu-semtech/mu-authorization)
+- [delta-notifier](https://github.com/mu-semtech/delta-notifier)
+
+### Adding the service to your stack
+
+Add the following snippet to your `docker-compose.yml` to include this
+service in your project.
+
+```yaml
+version: '3.4'
+
+services:
+  virus-scanner:
+    build: https://github.com/peternowee/virus-scanner-service.git#v0.0.8
+    links:
+      - database:database
+    environment:
+      VIRUS_SCANNER_CLAMD_USER: # "root"
+    volumes:
+      - ./data/files:/share
+      - type: volume
+        source: virus-scanner-signatures
+        target: /var/lib/clamav
+
+volumes:
+  virus-scanner-signatures:
+```
+
+> Note: The ClamAV authors do not recommend running `clamd` as `root` for
+> safety reasons because ClamAV scans untrusted files that may be
+> malware. However, the file-service currently saves its files with
+> access permission for `root` only. Consider the security implications
+> for your situation before uncommenting the line to let `clamd` run as
+> `root`:
+>
+>       VIRUS_SCANNER_CLAMD_USER: "root"
+
+
+Add rules to `dispatcher.ex` to dispatch requests to this service. E.g.
```

Comment:
Your text with improved grammar and spelling:

"General: Given the current scope, I am very much inclined to keep the API in the documentation for 'debugging' purposes only. Although it could be potentially useful, it needs thorough consideration before making it available to production clients (mainly the frontend). I do not consider this as part of the scope for this iteration."

Reply:
Given the current scope, I am very much inclined to keep the API in the documentation for 'debugging' purposes only. Although it could be potentially useful, it needs thorough consideration before making it available to production clients (mainly the frontend). I do not consider this as part of the scope for this iteration."